x/ccv/provider/keeper/consumer_equivocation.go

```diff
@@ -390,19 +393,21 @@ func (k Keeper) ComputePowerToSlash(ctx sdk.Context, validator stakingtypes.Vali
 	redelegations []stakingtypes.Redelegation, power int64, powerReduction math.Int,
 ) int64 {
 	// compute the total numbers of tokens currently being undelegated
-	undelegationsInTokens := sdk.NewInt(0)
+	undelegationsInTokens := math.NewInt(0)
 
 	// Note that we use a **cached** context to avoid any actual slashing of undelegations or redelegations.
 	cachedCtx, _ := ctx.CacheContext()
 	for _, u := range undelegations {
-		amountSlashed := k.stakingKeeper.SlashUnbondingDelegation(cachedCtx, u, 0, sdk.NewDec(1))
+		// v50: errors are ignored
```

Comment:
Why are the errors ignored?

Reply:
There's a note in the function comment:
```go
// Note that this method does not perform any slashing.	
```

Panicking seems unecessary, and you can't do anything useful with the error.

@sainoe Any suggestions for refactoring this? This code is new to me.